### ISSUE TITLE
Updates for Russian translation

### ISSUE THREAD

**Issue by Totktonada** [weechat/weechat#304]
In gist revision 5: https://gist.github.com/Totktonada/3caa075f886c75f4aabc

Thanks for mention me in translators! :)

**Comment by flashcode** [comment]
Thanks, doc is updated.
One translation is missing in Russian man page, the line with your name.

**Comment by Totktonada** [comment]
```
-// TRANSLATION MISSING
-Translated by Alexander Turenko <totktonada.ru@gmail.com>.
+Перевод Александра Туренко <totktonada.ru@gmail.com>.
```

And we done this step :)

**Comment by Totktonada** [comment]
tl;dr: I prefer latinized name “Alexander Turenko” for recognizability, but when it used in non-root form better to declention it in cyrillic form as appropriate. So if you want to unify name form (it okay for me too), point it in root form everywhere. I think that is better appoach, just as in quickstart: `Translator(s): name of list here`.

«Александр**а** Туренко» isn’t root form of noun. Root form: «Александр Туренко», but when “translation of” then «Александра Туренко». If you want go deeper to Russian names, then «Александр» is root form for *male* name (my case :)), but «Александра» is root form of *female* name and *non-root form of male* name.

**Comment by Totktonada** [comment]
I don’t know how to describe Russian words flexibility in English terms, so if you don’t understand second paragraph of prev message entirely, it’s okay.

**Comment by flashcode** [comment]
I fixed the name, but not sure if that's what you suggested :p

**Comment by Totktonada** [comment]
Thanks! Current formatting is better :)

For better punctuation you can add colon after word «Перевод», than:
```
Перевод: Alexander Turenko <totktonada.ru@gmail.com>.
```

That’s all.
